### PR TITLE
CV

### PR DESCRIPTION
**Pull request by sanisoclem**

### CODE REVIEW

File: _data/resume.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    "label": "Programmer",
     "picture": "",
-    "email": "",
-    "phone": "",
-    "website": "",
-    "summary": "",
+    "email": "contact@jerahmeelcosinas.net",
+    "phone": "+65 9026 7517",
+    "website": "https://jerahmeelcosinas.net",
+    "summary": "Whatcha gonna do with all that shit, all that shit inside your pwit",
```
Comment (joelleortiz) [inline _data/resume.json]: Inconsistent spelling

File: _sass/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  
+  &:hover {
+    text-decoration: none;
+  }
+  
+  @include breakpoint('md') {
+    width:auto;
+    display: inline-block;
+    padding: 0.6rem 0.9rem;
+    font-size: 0.9rem;
```
Comment (joelleortiz) [inline _sass/_buttons.scss]: Duplicate style

File: _sass/_vanilla.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// Override default Vanilla settings in your main Sass file
+$font-base-family: 'Merriweather';
```
Comment (joelleortiz) [inline _sass/_vanilla.scss]: Unnecessary file

File: _data/resume.json (hunk 1 above)
Comment (sanisoclem) [reply _data/resume.json]:
```suggestion
    "summary": "Watcha gonna do with all that shit, all that shit inside your pwit",
```

**Pull request merged**